.github/workflows/save-send-runtime-binary.yml

```diff
@@ -0,0 +1,83 @@
+name: Save send-runtime binary as an artifact
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'local-tests/send-runtime/**'
+
+jobs:
+  build:
+    name: Save send-runtime binary as an artifact
+    runs-on: ubuntu-latest
+    env:
+      RUST_TOOLCHAIN_VERSION: nightly-2021-10-24
+      CARGO_INCREMENTAL: 0
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout source code
+        uses: actions/checkout@v2
+
+      - name: GIT | Get branch info & current commit sha.
+        id: vars
+        shell: bash
+        run: |
+          echo "##[set-output name=branch;]$(echo ${GITHUB_REF##*/})"
+          echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+
+      - name: Print toolchain version
+        run: echo running with "$RUST_TOOLCHAIN_VERSION"
+
+      - name: Install rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: ${{ env.RUST_TOOLCHAIN_VERSION }}
+          override: true
+
+      - name: Build binary
+        run: |
+          pushd local-tests/send-runtime/
+          cargo build --release
+          popd
+
+      - name: Upload binary
```

Comment:
Why are we uploading the "bare" binary as an artifact?

Reply:
leftover from previous approach; removed